phoebe/atmospheres/passbands.py

```diff
@@ -310,7 +310,7 @@ def save(self, archive, overwrite=True, update_timestamp=True, history_entry='')
         data.append(primary_hdu)
 
         # Tables:
-        ptf_table = Table(self.ptf_table)
```

Comment:
was this just completely unused in this method?

Reply:
yep, completely unused.